syntax = "proto3";

package playground;

service LocustService {
  rpc RunTest(stream TestRequest) returns (stream TestResponse) {}
}

message TestRequest {
  int32 test_id = 1;
  string name = 2;
}

message TestResponse {
  int32 test_id = 1;
  string status = 2;
}
